SalesforceCommerceCloud/pwa-kit
# Set Input Modes for Address/Payment Forms

## Problem

On mobile the wrong keyboard type is showing for various form fields. This PR corrects them using the `inputmode` and in some parts the `type` attributes.

 **GUS**: [W-9455333](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000OOm6YAG/view)
 **Linked PRs**: (links to corresponding PRs, optional)

## How to test-drive this PR
- On mobile (using your ios simulator) check that the correct keyboards are being shown for:
- Guest Checkout Payments and Address forms
- Registered User checkout for payment and address forms
- Registered User profile adding for payments and addresses.

## Changes
- Use numeric input mode and type tel where required.
